meson.build

```diff
@@ -0,0 +1,260 @@
+  'src/winscard_msg_srv.c',
+  'src/winscard_svc.c'
+  ]
+gen_src = gen_flex.process('src/configfile.l', 'src/tokenparser.l')
+pcscd_src += gen_src
+incdir = include_directories(['src', 'src/PCSC'])
+
+# dependencies
+threads_dep = dependency('threads')
+pcscd_dep = [threads_dep]
+if get_option('libudev')
+  udev_dep = dependency('libudev')
+  pcscd_dep += udev_dep
+  conf_data.set('HAVE_LIBUDEV', true)
+  features += 'udev'
+endif
+
+if get_option('libusb')
+  libusb_dep = dependency('libusb-1.0')
+  pcscd_dep += libusb_dep
+  conf_data.set('HAVE_LIBUSB', true)
+  features += 'libusb'
+endif
+
+polkit_dep = dependency('polkit-gobject-1', required : get_option('polkit'))
+if polkit_dep.found()
+  conf_data.set('HAVE_POLKIT', true)
+  features += 'polit'
+endif
+
+systemd_dep = dependency('libsystemd', required : get_option('libsystemd'))
+if systemd_dep.found()
+  features += 'systemd'
+endif
+
+if target_machine.system() == 'darwin'
+  pcscd_dep += dependency('appleframeworks', modules : ['foundation', 'IOKit'])
+  pcscd_src += files(['src/hotplug_macosx.c', 'src/dyn_macosx.c'])
+  conf_data.set_quoted('PCSC_ARCH', 'MacOS')
+elif target_machine.system() == 'linux'
+  conf_data.set_quoted('PCSC_ARCH', 'Linux')
+elif target_machine.system() == 'netbsd'
+  conf_data.set_quoted('PCSC_ARCH', 'NetBSD')
+endif
+features +=  conf_data.get_unquoted('PCSC_ARCH')
+pcscd_dep += [polkit_dep, systemd_dep]
+
+# end of PCSCLITE_FEATURES
+conf_data.set('PCSCLITE_FEATURES', '"' + ' '.join(features) + '"')
+
+executable('pcscd',
+  sources : pcscd_src,
+  include_directories : incdir,
+  dependencies : pcscd_dep,
+  c_args: '-DPCSCD',
+  export_dynamic : true,
+  install_dir : '/' / get_option('sbindir'),
+  install : true)
+
+# libpcsclite_real library
+libpcsclite_real_src = [
+  'src/debug.c',
+  'src/winscard_clnt.c',
+  'src/simclist.c',
+  'src/sys_unix.c',
+  'src/utils.c',
+  'src/winscard_msg.c'
+  ]
+library('pcsclite_real',
+  libpcsclite_real_src,
+  include_directories : incdir,
+  dependencies : threads_dep,
+  c_args: '-DLIBPCSCLITE -DSIMCLIST_NO_DUMPRESTORE',
+  soversion : 1,
+  install : true)
+
+# libpcsclite library
+libpcsclite_src = [
+  'src/error.c',
+  'src/libredirect.c',
+  'src/sys_unix.c'
+  ]
+libpcsclite = library('pcsclite',
+  libpcsclite_src,
+  include_directories : incdir,
+  soversion : 1,
+  install : true)
+
+# libpcsclite_fake library
+library('pcsclite_fake',
+  sources : 'src/libfake.c',
+  include_directories : incdir)
+
+# libpcscspy library
+library('pcscspy',
+  sources : ['src/spy/libpcscspy.c', 'src/sys_unix.c'],
+  include_directories : incdir,
+  soversion : 0,
+  install : true)
+
+install_data('src/spy/pcsc-spy',
+  install_dir : get_option('bindir'))
+
+install_data('src/spy/setup_spy.sh',
+  install_dir : 'share/doc/pcsc-lite')
+
+run_command('pod2man',
+  ['--date=2024-01-01', 'src/spy/pcsc-spy.pod', 'pcsc-spy.1'],
+  check : true)
+install_data('pcsc-spy.1',
+  install_dir : get_option('mandir') + '/man1')
```

Comment:
nit: `join_paths` is more portable when merging paths than simple concatenation.

Reply:
Done